src/gwc/pom.xml

```diff
@@ -46,12 +46,6 @@
     <dependency>
       <groupId>org.geowebcache</groupId>
       <artifactId>gwc-georss</artifactId>
-      <exclusions>
-        <exclusion>
-           <groupId>com.vividsolutions</groupId>
-           <artifactId>jts</artifactId>
-        </exclusion>
-      </exclusions>
```

Comment:
you may want to keep the exclusion if geowebcache gets on a different version of jts then geoserver?